packages/image_picker/lib/image_picker.dart

```diff
@@ -58,13 +58,15 @@ class ImagePicker {
 
   static Future<File> pickVideo({
     @required ImageSource source,
+    double maxDuration
```

Comment:
Can you add a doc comment to pickVideo that documents this function (similarly to how it is done for pickImage) and also describe the new parameter `maxDuration` in there?